src/ex6/server/client/src/components/AddTask/AddTask.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from "react"
+import "./style.css"
+import { ItemClient } from "./item_client"
+import deleteIcon from "./images/delete_icon.svg"
+
+export const AddTask = () => {
+  const itemClient = new ItemClient()
```

Comment:
use semicolons ; at the end of each statement - this is a styling that persists across most JS projects these days